code/modules/reagents/reagent_containers/food/condiment.dm

```diff
@@ -148,6 +150,146 @@
+	fixed_state = TRUE
+	name = "berry syrup"
+	reagents_to_add = list(/singleton/reagent/condiment/syrup_berry = 50)
+
+/obj/item/reagent_containers/food/condiment/syrup_raspberry
+	icon_state = "syrup_raspberry"
+	fixed_state = TRUE
+	name = "raspberry syrup"
+	reagents_to_add = list(/singleton/reagent/condiment/syrup_raspberry = 50)
+
+/obj/item/reagent_containers/food/condiment/syrup_blueberry
+	icon_state = "syrup_blueberry"
+	fixed_state = TRUE
+	name = "blueberry syrup"
+	reagents_to_add = list(/singleton/reagent/condiment/syrup_blueberry = 50)
+
+/obj/item/reagent_containers/food/condiment/syrup_blue_raspberry
+	icon_state = "syrup_blue_raspberry"
+	fixed_state = TRUE
+	name = "blue raspberry syrup"
+	reagents_to_add = list(/singleton/reagent/condiment/syrup_blueberry = 50)
+
+/obj/item/reagent_containers/food/condiment/syrup_ylphaberry
+	icon_state = "syrup_ylpha"
+	fixed_state = TRUE
+	name = "ylpha berry syrup"
+	reagents_to_add = list(/singleton/reagent/condiment/syrup_ylphaberry = 50)
+
+/obj/item/reagent_containers/food/condiment/syrup_caramel
+	icon_state = "syrup_caramel"
+	fixed_state = TRUE
+	name = "caramel syrup"
+	reagents_to_add = list(/singleton/reagent/condiment/syrup_caramel = 50)
+
+/obj/item/reagent_containers/food/condiment/syrup_pumpkin
+	icon_state = "syrup_pumpkin"
+	fixed_state = TRUE
+	name = "pumpkin syrup"
+	reagents_to_add = list(/singleton/reagent/condiment/syrup_pumpkin = 50)
+
+/obj/item/reagent_containers/food/condiment/syrup_vanilla
+	icon_state = "syrup_vanilla"
+	fixed_state = TRUE
+	name = "vanilla syrup"
+	reagents_to_add = list(/singleton/reagent/condiment/syrup_vanilla = 50)
+
+/obj/item/reagent_containers/food/condiment/syrup_dirtberry
+	icon_state = "syrup_dirtberry"
+	fixed_state = TRUE
+	name = "dirtberry syrup"
+	reagents_to_add = list(/singleton/reagent/condiment/syrup_dirtberry = 50)
+
+/obj/item/reagent_containers/food/condiment/gelatin
+	icon_state = "gello"
+	fixed_state = TRUE
+	name = "gelatin"
+	reagents_to_add = list(/singleton/reagent/nutriment/gelatin = 50)
+
+/obj/item/reagent_containers/food/condiment/batter
+	icon_state = "batter"
+	fixed_state = TRUE
+	name = "batter mix"
+	reagents_to_add = list(/singleton/reagent/nutriment/coating/batter = 50)
+
+/obj/item/reagent_containers/food/condiment/vanilla
+	icon_state = "vanilla"
+	fixed_state = TRUE
+	name = "vanilla extract"
+	reagents_to_add = list(/singleton/reagent/nutriment/vanilla = 50)
+
+/obj/item/reagent_containers/food/condiment/rice //everyone's favorite condiment, rice. (there wasn't really a better place for this, code-wise)
+	icon_state = "rice"
+	fixed_state = TRUE
+	name = "rice"
+	reagents_to_add = list(/singleton/reagent/nutriment/rice = 50)
+
+/obj/item/reagent_containers/food/condiment/coco //not exactly a condiment, but not exactly NOT a condiment, right?
+	icon_state = "cocoapowder"
+	fixed_state = TRUE
+	name = "coco powder"
+	reagents_to_add = list(/singleton/reagent/nutriment/coco = 50)
```

Comment:
```suggestion
/obj/item/reagent_containers/food/condiment/cocoa //not exactly a condiment, but not exactly NOT a condiment, right?
	icon_state = "cocoapowder"
	fixed_state = TRUE
	name = "cocoa powder"
	reagents_to_add = list(/singleton/reagent/nutriment/coco = 50)
```
you don't have to touch the reagent since you didn't add it.